src/core/default-modules.ts

```diff
@@ -1,45 +1,85 @@
+} from '../modules/ui/core/theme';
 import TextMap from '../modules/text-map/text-map';
 
-import FullScreenManager from '../modules/full-screen-manager/full-screen-manager';
+import FullScreenManager, {
+  IPlayer as IFullScreenManagerPlayer,
+} from '../modules/full-screen-manager/full-screen-manager';
 
 import LiveStateEngine from '../modules/playback-engine/live-state-engine';
 import KeyboardControls from '../modules/keyboard-control/keyboard-control';
 import DebugPanel from '../modules/ui/debug-panel/debug-panel';
 
-import Screen from '../modules/ui/screen/screen';
+import Screen, { IPlayer as IScreenPlayer } from '../modules/ui/screen/screen';
 import InteractionIndicator from '../modules/ui/interaction-indicator/interaction-indicator';
 
-import Overlay from '../modules/ui/overlay/overlay';
+import Overlay, {
+  IPlayer as IOverlayPlayer,
+} from '../modules/ui/overlay/overlay';
 import Loader from '../modules/ui/loader/loader';
 
-import MainUIBlock from '../modules/ui/main-ui-block/main-ui-block';
-import TopBlock from '../modules/ui/top-block/top-block';
-import Title from '../modules/ui/title/title';
+import MainUIBlock, {
+  IPlayer as IMainUIBlockPlayer,
+} from '../modules/ui/main-ui-block/main-ui-block';
+import TopBlock, {
+  IPlayer as ITopBlockPlayer,
+} from '../modules/ui/top-block/top-block';
+import Title, { IPlayer as ITitlePlayer } from '../modules/ui/title/title';
 import LiveIndicator from '../modules/ui/live-indicator/live-indicator';
 
-import BottomBlock from '../modules/ui/bottom-block/bottom-block';
-import ProgressControl from '../modules/ui/controls/progress/progress';
+import BottomBlock, {
+  IPlayer as IBottomBlockPlayer,
+} from '../modules/ui/bottom-block/bottom-block';
+import ProgressControl, {
+  IPlayer as IProgressControlPlayer,
+} from '../modules/ui/controls/progress/progress';
 import PlayControl from '../modules/ui/controls/play/play';
 import TimeControl from '../modules/ui/controls/time/time';
 import VolumeControl from '../modules/ui/controls/volume/volume';
 import FullScreenControl from '../modules/ui/controls/full-screen/full-screen';
-import Logo from '../modules/ui/controls/logo/logo';
-import DownloadButton from '../modules/ui/controls/download/download';
-
-import PreviewService from '../modules/ui/preview-service/preview-service';
+import Logo, { IPlayer as ILogoPlayer } from '../modules/ui/controls/logo/logo';
+import DownloadButton, {
+  IPlayer as IDownloadButtonPlayer,
+} from '../modules/ui/controls/download/download';
+
+import PreviewService, {
+  IPlayer as IPreviewServicePlayer,
+} from '../modules/ui/preview-service/preview-service';
 import PreviewThumbnail from '../modules/ui/preview-thumbnail/preview-thumbnail';
 import PreviewFullSize from '../modules/ui/preview-full-size/preview-full-size';
 
 import { TooltipService } from '../modules/ui/core/tooltip';
 
 const { asClass } = DependencyContainer;
 
+export type IPlayer = IRootContainerPlayer &
```

Comment:
Not sure, you should name type starting with `I`.

Reply:
We have a convention. All types should start from `I` (declared by `type` and `interface).